### PR TITLE
subscription: don't assume muted tracks stalled

### PR DESCRIPTION
**Pull request by daniel-abramov**
When the user mutes the video, we don't get any RTPs. We don't assume this error anymore and don't change the status of the track and its telemetry for that.

This removes false positives related to the
https://github.com/matrix-org/waterfall/issues/121

### CODE REVIEW

File: pkg/conference/participant/tracker.go [hunk 1]
```diff
@@ -161,7 +161,11 @@ func (t *Tracker) RemovePublishedTrack(id track.TrackID) {
 }
 
 // Subscribes a given participant to the track.
-func (t *Tracker) Subscribe(participantID ID, trackID track.TrackID, requirements track.TrackMetadata) error {
+func (t *Tracker) Subscribe(
+	participantID ID,
+	trackID track.TrackID,
+	desiredWidth, desiredHeight int,
+) error {
```
Comment (dbkr) [inline pkg/conference/participant/tracker.go:168]: This change doesn't seem related?

**Review by dbkr — APPROVED** [review]
Otherwise lgtm

File: pkg/conference/participant/tracker.go (hunk 1 above)
Comment (daniel-abramov) [reply pkg/conference/participant/tracker.go:168]: I changed the argument from `TrackMetadata` to `desiredWidth, desiredHeight`. Previously these were the only fields in the `TrackMetadata`. Now, it's more extensive (contains the mute state and probably will contain more), i.e. it was required since I changed the input parameters for the `published.Subscribe()` function.
